lib/MusicBrainz/Server/Controller/Work.pm

```diff
@@ -186,6 +186,12 @@ after create => sub {
     stash_work_form_json($c);
 };
 
+before qw( create edit ) => sub {
+    my ($self, $c) = @_;
+    my %work_types = map {$_->id => $_} $c->model('WorkType')->get_all();
+    $c->stash->{work_types} = \%work_types
```

Comment:
Nitpick: add a semicolon at the end of the line? :)

Reply:
Darn.